ocaml/otherlibs/systhreads/st_posix.h

```diff
@@ -269,6 +269,20 @@ Caml_inline void st_thread_yield(st_masterlock * m)
   pthread_mutex_unlock(&m->lock);
 }
 
+Caml_inline void st_fallback_yield()
```

Comment:
I'm a little worried about the reappearance of this code, since it had problems with fairness: https://github.com/ocaml/ocaml/pull/2112. IIUC it won't get used with the default locking scheme, but are we sure we want to use it with the python locking schemes. Even if we are, can we find a way to put this code into the place that defines the python scheme rather than having a fallback in the runtime.

Reply:
There aren't really any other options! Trying to do busy-waiting yielding between threads efficiently with Python in the mix is basically not going to work.